appinventor/components/src/com/google/appinventor/components/runtime/Slider.java

```diff
@@ -264,7 +264,7 @@ public float MinValue() {
   @DesignerProperty(editorType = PropertyTypeConstants.PROPERTY_TYPE_FLOAT,
       defaultValue = Component.SLIDER_MAX_VALUE + "")
   @SimpleProperty(description = "Sets the maximum value of slider.  Changing the maximum value " +
-      "also resets Thumbposition to be halfway between the minimum and the (new) maximum. " +
+      "also resets the thumb position to be halfway between the minimum and the (new) maximum. " +
```

Comment:
ThumbPosition is a property; the only change needed here would be a capital P --> ` also resets ThumbPosition to be halfway...`